pkg/virtctl/create/params/params.go

```diff
@@ -57,6 +57,8 @@ func Supported(obj interface{}) string {
 			t = structField.Type.Elem().String()
 		case structField.Type == reflect.TypeOf(&resource.Quantity{}):
 			t = structField.Type.Elem().String()
+		case structField.Type.Kind() == reflect.Slice:
```

Comment:
Is this for parsing string slices? You should make sure it is a string slice and not only a slice.